### PR TITLE
tests: part 2 support for VK_EXT_shader_tile_image

### LINKED ISSUE #5630: Add VK_EXT_shader_tile_image shader validation
-  [ ] VUID-RuntimeSpirv-shaderTileImageColorReadAccess-08728
-  [ ] VUID-RuntimeSpirv-shaderTileImageDepthReadAccess-08729
-  [ ] VUID-RuntimeSpirv-shaderTileImageStencilReadAccess-08730
-  [ ] VUID-RuntimeSpirv-minSampleShading-08731
-  [ ] VUID-RuntimeSpirv-minSampleShading-08732

(also for `spirv-val` the following are needed)
-  VUID-StandaloneSpirv-None-08720
-  VUID-StandaloneSpirv-TileImageEXT-08723
-  VUID-StandaloneSpirv-None-08724

### PR DESCRIPTION
**Pull request by mohfai-arm**
closes #5630

### CODE REVIEW

**Comment by janharaldfredriksen-arm** [comment]
Note: This will depend on the changes in #5736 and likely fails as a standalone patch, but we wanted to get everything up for review.

File: tests/negative/dynamic_rendering.cpp [hunk 1]
```diff
@@ -221,6 +221,157 @@ TEST_F(VkLayerTest, DynamicRenderingCommandDraw) {
...
+    char const *fsSource = R"glsl(
+            #version 450
+            #extension GL_EXT_shader_tile_image : require
+            layout(early_fragment_tests) in;
+            layout(location = 0) out vec4 uFragColor;
+            void main() {
+                highp float depth = depthAttachmentReadEXT();
+                uint stencil = stencilAttachmentReadEXT();
+                uFragColor = vec4(depth,float(stencil),0.0f,1.0f);
+            }
```
Comment (ncesario-lunarg) [inline tests/negative/dynamic_rendering.cpp:272]: @mohfai-arm if this is blocked by glslang changes, you can also just paste the SPIR-V assembly here and set the `source_type` parameter to `SPV_SOURCE_ASM` when creating the the `VkShaderObj` object.

**Review by ncesario-lunarg — CHANGES_REQUESTED** [review]
LGTM, but a positive test utilizing `VK_EXT_shader_tile_image` should also be added.

**Comment by mohfai-arm** [comment]
closing this PR because tests are moved into #5736 with spirv

**Pull request closed without merging**